### DISCUSSION TITLE
Using React-Intl in Head - NextJS 13 (app folder)

### DISCUSSION THREAD

**Discussion by st11x** [vercel/next.js#44247] [Help]
Hi,

We are using the new app folder and have created a number of head.tsx file for our routes. 
When trying to use the react-intl library, it fails to find the IntlProvider because that is defined in the top level layout.tsx and it seems like head.tsx is rendered separate from it. 

We are just trying to translate the page title. 
  
Thanks
Matt

**Comment by JustFly1984** [comment]
I have same issue with new Gatsby.js Head API not wrapped in Providers too. Looks like switching to Next.js is not a solution for my app issues with gatsby.js

Internationalization is complex by itself. It is sad that new features ignore backward compatibilities to ensure internationalization is not broken.
In gatsby.js issues I was straight asked to make it myself... It is understandable why their community is dying in compare with next.js and other newest frameworks.
